<launch>

  <include file="$(find duo3d_ros)/launch/duo_proc.launch" />

  <node name="vio_ros_node" pkg="vio_ros" type="vio_ros_node" output="screen" >
    <remap from="/imu" to="/duo3d_camera/cam_imu"/>
    <remap from="/left_image" to="/left/image_rect"/>
    <remap from="/right_image" to="/right/image_rect"/>
  </node>
</launch>
